package.json

```diff
@@ -61,8 +61,10 @@
     "deep-extend": "0.6.0",
     "fast-xml-parser": "3.19.0",
     "form-data": "3.0.0",
+    "fuzzy": "^0.1.3",
     "glob": "7.1.4",
     "inquirer": "8.2.0",
+    "inquirer-checkbox-plus-prompt": "^1.0.1",
```

Comment:
another concern Tian pointed out is how do we know this package is reliable? It solves the issue but i'm not sure how to decide if it's reliable or not

Reply:
There are a few key things I look for:
1. Is the API stable? If it's > 1.x, the answer is usually `yes`.
2. Is this core to my functionality? If not, can we do it ourself?
3. Is the project active? If it's not, are we comfortable owning our use of it in our application or library?
4. Does it have any dependencies? If so, are we comfortable taking them on?
5. What's the license? Is it compatible with the license we use?

In this case, the `inquirer-checkbox-plus-prompt` is pretty simple, around 500 lines of code. It's not actively updated, but it might literally just be `done` and not need any updates.

I'd investigate if we could take just the methods you need and add them (with attribution, of course, if the license allows).